### ISSUE TITLE
How to change the learning rate after restore_from_checkpoint?

### ISSUE THREAD

**Issue by newoneincntk** [Microsoft/CNTK#2407]
How to change the learning rate after restore_from_checkpoint?

**Comment by cha-zhang** [comment]
learner.reset_learning_rate should work.

**Comment by newoneincntk** [comment]
Thank you very much for your reply. I made it in the following way by defining a new learner, was it the same as learner.reset_learning_rate?

   if is_retraining:
       gtrainer.restore_from_checkpoint('Models/LPSCNN_{}.dnn'.format(14))
    learner1 = C.learners.adam(z.parameters, lr_schedule, mm_schedule, minibatch_size=minibatch_size)
    trainer = C.Trainer(gtrainer.model, (ce, pe), learner1, progress_printer)

**Comment by eldakms** [comment]
Not really, you will lose the state of you learner and trainer.
It should be something like this:

trainer.restore_from_checkpoint(...);
// you have a single learner right?
**trainer.parameter_learners[0].reset_learning_rate(whatever value you need)**
// use your trainer here.

**Comment by cha-zhang** [comment]
Hope this issue has been resolved. Close it now. Feel free to reopen.
